### ISSUE TITLE
Option to delete only non-public toots

### ISSUE THREAD

**Issue by rugk** [codl/forget#219]
Use case: My non-public toots may contain a little more personal data than my public ones. Thus, I want to archive my public toots.

Also previously requested [in here](https://social.wiuwiu.de/web/statuses/102146546715619278). (see also the example script that has been posted)

**Comment by rugk** [comment]
shh #badbot :no_bell::robot::no_bell: This is a feature I'd still like.

**Comment by codl** [comment]
Yeah, I would like to implement this when I can. I think it's a good heuristic

**Comment by DC7IA** [comment]
I have the opposite use case to @rugk's: I'd like to online have public toots deleted. :D
